docs/generate.sh

```diff
@@ -2,12 +2,16 @@
 
 set -e
 
-FOLDER=docs/source/demos/
+FOLDER=docs/source/demos
 
 for file in "${FOLDER}/01_linear_function" "${FOLDER}/02_multilinear_function" "${FOLDER}/03_nonlinear_function"; do
 	jupytext --to ipynb "${file}.py"
     jupyter-nbconvert --to markdown --execute --ExecutePreprocessor.timeout=None "${file}.ipynb"
 done
 
+FOLDER=docs/source/jaxducc
+jupytext --to ipynb "docs/source/contrib/jaxducc0.py"
```

Comment:
I think you forgot to link `jaxducc0` into "docs/source/contrib". As is, the documentation does not build for me.

Reply:
Ah thanks, I had not added the docs/sources/contrib folder to git before pushing